Algorithms/Algorithms.Tests/ReversePolishNotationTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+using System;
+using Xunit;
+
+namespace Algorithms.Tests
+{
+    public class ReversePolishNotationTests
+    {
+        [Theory]
+        [InlineData("1+1", 2)]
```

Comment:
May need more examples with spaces and without.
For example:
2+2*2 and 2 +2* 2
can that example pass tests?
And for every token.

Reply:
And check negative numbers